### ISSUE TITLE
Docs: Write three guides for setting up long-term storage backend, using Helm

### ISSUE THREAD

**Issue by osg-grafana** [grafana/mimir#4458]
- #4459
- #4460
- #4461

**Comment by osg-grafana** [comment]
@eamonryan and @krajorama, are you two still intending to write this content. If not, please unassign yourselves.

**Comment by eamonryan** [comment]
@osg-grafana Yes I'd still like to write this - I want to commit the sample YAMLs to the repo also firstly, then reference them. Will try to squeeze it in soon.

**Comment by osg-grafana** [comment]
> @osg-grafana Yes I'd still like to write this - I want to commit the sample YAMLs to the repo also firstly, then reference them. Will try to squeeze it in soon.

I have added `FY24Q4` based on your feedback. If you need to push this out, please change the milestone to `FY25Q1`.
